transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/LibraryTransformations.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.monster.encode.transformation
+
+import java.time.OffsetDateTime
+
+import org.broadinstitute.monster.encode.jadeschema.table.Library
+import upack.Msg
+
+/** Transformation logic for ENCODE library objects. */
+object LibraryTransformations {
+  import org.broadinstitute.monster.common.msg.MsgOps
+
+  /** Transform a raw ENCODE library into our preferred schema. */
+  def transformLibrary(libraryInput: Msg): Library = {
+    Library(
+      id = CommonTransformations.readId(libraryInput),
+      crossReferences = libraryInput.read[Array[String]]("dbxrefs"),
+      timeCreated = libraryInput.read[OffsetDateTime]("date_created"),
+      award = libraryInput.read[String]("award"),
+      lab = libraryInput.read[String]("lab"),
+      queriedRnpSizeRange = libraryInput.tryRead[String]("queried_RNP_size_range"),
+      rnaIntegrityNumber = libraryInput.tryRead[Double]("rna_integrity_number"),
+      sizeRange = libraryInput.tryRead[String]("size_range"),
+      strandSpecificity = libraryInput.read[Boolean]("strand_specificity"),
+      treatments = libraryInput.read[Array[String]]("treatments"),
```

Comment:
Do we trust that this array will always be populated in the source data?

Reply:
From what I've seen, the extracted json always has `"treatments": []` when there are no treatments. I haven't seen any cases where the field is not defined, if that's what you're asking. We could change this to a `tryRead`, but would need to update it elsewhere too (we are using `read` to extract arrays of strings in the other transformations).

Reply:
I'm ok leaving it until we see a problem in practice